css/modules/styles.css

```diff
@@ -4135,10 +3910,14 @@ span#usertab_text {
 }
 
 #virtualclassCont.congrea #joinClass #joinClassModal .modal-dialog {
-  margin: 20% auto;
   width: 400px;
 }
 
+#virtualclassCont.congrea #joinClass #joinClassModal .modal-dialog .modal-content {
+  border-radius: 12px;
+  box-shadow: 0 2px 8px 0 rgba(32,33,36,0.28) !important;
```

Comment:
Codacy found an issue: [Use of !important](https://app.codacy.com/gh/congrea/virtualclass/pullRequest?prid=5740126)